src/Entity/User.php

```diff
@@ -22,7 +22,7 @@ class User extends BaseUser
      * @ORM\GeneratedValue()
      * @ORM\Column(type="integer")
      */
-    private $id;
```

Comment:
I doubt this is needed, you should copy this whole class when making an extend

Reply:
Yeah, but since it is an ORM column I think it should be protected anyway. Otherwise this class won't work.